README.md

```diff
@@ -32,7 +32,24 @@ Coming soon.
 Testing
 -------
 
-Coming soon.
+To manually run the tests with your current version of Python (system or virtual environment):
+
+1. Install the test Requirements:
+       pip install -r requirements/requirements-test.txt
```

Comment:
```suggestion
       pip install -r requirements.txt
```

Reply:
my requirements files got messy there at the end.... when I wrote this, it _did_ install Django, but not anymore